lib/screens/surveydetails/widget/answer_multi_choice_widget.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:lydiaryanfluttersurvey/model/ui/question_ui_model.dart';
+import 'package:lydiaryanfluttersurvey/resources/dimensions.dart';
+
+class AnswerMultiChoiceWidget extends StatefulWidget {
+  final QuestionUiModel question;
+  final Function(List<String>) onChecked;
+
+  const AnswerMultiChoiceWidget({
+    Key? key,
+    required this.question,
+    required this.onChecked,
+  }) : super(key: key);
+
+  @override
+  State<AnswerMultiChoiceWidget> createState() =>
+      _AnswerMultiChoiceWidgetState();
+}
+
+class _AnswerMultiChoiceWidgetState extends State<AnswerMultiChoiceWidget> {
+  final List<String> _answers = [];
+  final List<int> _checkedIndex = [];
```

Comment:
I think we can simplify a little bit 😇 
What if instead of maintaining two lists, instead we maintain a set of IDs instead? 😇 
```suggestion
  final Set<String> _checkedAnswerIds = {};
```

Reply:
Updated all in a68abb79